atlas/strategies/randomized.py

```diff
@@ -46,4 +49,9 @@ def Sequence(self, domain: Any, context: Any = None, max_len: int = None,
             lengths = range(1, max_len + 1)
 
         return random.choices(domain, k=random.choice(lengths))
-    
+
+    @operator
```

Comment:
Let's keep `SequenceFixed` and `SelectFixed` inside the pandas instantiation as they don't make much sense when they are not tied to the model. Same goes for the DfsStrategy.